luis-eduardo-salcedo/Tarea-4/js/code.js

```diff
@@ -0,0 +1,93 @@
+
+let valorActual = 0; 
+let operacion = "0"; 
+
+let valorAnterior = null;
+
+const calculadoraScreen = document.querySelector(".section-botones__input-numeros");
```

Comment:
@luisSalcedo8 , please use English as primary language to name funcions and variables